back/engines/commercial/project_management/spec/mailers/email_campaigns/project_moderation_rights_received_mailer_spec.rb

```diff
@@ -1,6 +1,10 @@
+# frozen_string_literal: true
+
 require 'rails_helper'
 
-RSpec.describe EmailCampaigns::ProjectModerationRightsReceivedMailer, type: :mailer do
+skip_reason = defined?(EmailCampaigns::Engine) ? nil : 'email_campaigns engine is not installed'
```

Comment:
The email campaign engine was not really refactored (patched). We rely on the dummy predicate methods in user.rb. But some tests have been disabled.